modules/reference/pages/rpk/rpk-security/rpk-security-secret-create.adoc

```diff
@@ -0,0 +1,66 @@
+= rpk security secret create
+// tag::single-source[]
+
+Create a new secret for your Redpanda Cloud cluster.
```

Comment:
```suggestion
Create a new secret for your Redpanda Cloud cluster.
```
Will these commands ever be included in SM? If so, you might want to edit all of the commands from Redpanda Cloud cluster to just cluster.

Reply:
rpk security secret is Cloud only, but probably a good idea to keep it generic

Reply:
Updated